tests/test_soak.py

```diff
@@ -9,7 +9,7 @@
 import sdk_utils
 
 import spark_utils as utils
-from tests.test_kafka import KAFKA_PACKAGE_NAME, test_pipeline
+# from tests.test_kafka import KAFKA_PACKAGE_NAME, test_pipeline
```

Comment:
Let's rather remove this than comment it out.

Reply:
this was commented by mistake, reverted back now.